specification/_types/aggregations/Aggregate.ts

```diff
@@ -589,11 +597,12 @@ export class UnmappedSignificantTermsAggregate extends MultiBucketAggregateBase<
 /** @variant name=composite */
 // Note: no keyed variant
 export class CompositeAggregate extends MultiBucketAggregateBase<CompositeBucket> {
-  after_key?: Dictionary<string, UserDefinedValue>
+  // Must be consistent with CompositeAggregation.after and CompositeBucket.key
```

Comment:
This is great, would using an alias here make using this type easier for clients?

Reply:
There are two common patterns in ES to provide a "position" in a paginated/scrolled search/aggregation: either as a (field, value) dict as is the case here, or as an array of values, which we can see in `SortResults` but can also be found in many different places.

Not sure how to name those. Maybe something like:
- `type FieldValueDict = Dictionary<Field, FieldValue>`
- `type FieldValues = Array<FieldValue>`

Or a more abstract but also more semantic `IndexPositionDict` and `IndexPositionValues`?

Reply:
Good point, I tend to generalize a bit too much ;-) Added a commit with `CompositeAggregateKey`!